tests/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:13.0.1-alpine AS buildstep
+LABEL maintainer "RPS <devops@rpsgroup.com>"
+
+RUN mkdir -p /web
+WORKDIR /web
+
+COPY ./web/ /web/
+RUN yarn global add grunt-cli && \
+    yarn install && \
+    grunt
+
+FROM python:3.6
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir /glider-dac-status/logs
+COPY app.py config.yml flask_environments.py manage.py /glider-dac-status/
+COPY status /glider-dac-status/status
+COPY navo /glider-dac-status/navo
+COPY requirements/requirements.txt /requirements.txt
+COPY tests/test_status.py /glider-dac-status/
+RUN rm -f /glider-dac-status/test/test_status.py
```

Comment:
`test_status` is currently a de-facto smoke test -- it should pass.  More tests will be added to the module.  Is there a reason it's not included?

Reply:
It is run. It is copied from /test into the main directory first so that it can see all of its dependencies. I then remove it from /test because pytest doesn't like having two tests of the same name.